server/backend/api/app/crud/crud_user_role.py

```diff
@@ -145,6 +145,7 @@ def create(db: Session, user_id: int, role_id: int, requester: str):
     )
     db.add(new_fam_user_role)
     db.flush()
+    db.refresh(new_fam_user_role)
```

Comment:
Is this line forgotten for removal? I feel `.refresh` has no effect in the system.